snowflake_queries/account_admin_assigned.yml

```diff
@@ -0,0 +1,21 @@
+AnalysisType: scheduled_query
+QueryName: Snowflake.AccountAdminGranted
+Enabled: false
+Description: >
+  Monitor and detect granting account admin role
+SnowflakeQuery: >
+  --return instances where account admin role is granted
+  --this was adapted from a Security Feature Checklist query
+  SELECT 
+    user_name,
+    role_name,
+    query_text
+  FROM snowflake.account_usage.query_history
```

Comment:
Should this be time-bound to the last 60 minutes?

Reply:
Yes, the "look back" should match the schedule window.